test-basic/documents-quick.js

```diff
@@ -167,3 +167,67 @@ describe('quick path', function(){
     .catch(done);
   });
 });
+
+describe('test connection', function(){
+ var assert = require('assert');
+    it('should test the connection', function(done){
+     	var flag = false;
+     db.testConnection()
+      .result(function(response) {
+        flag = true;
```

Comment:
Now that test connection returns an object in all cases, would a better test assert that response.connected is true?